core/kazoo_bindings/src/kazoo_bindings.erl

```diff
@@ -710,7 +713,9 @@ map_processor_fold(#kz_binding{binding_parts=BParts
                   ,_Routing
                   ,RoutingParts
                   ) ->
-    case matches(BParts, RoutingParts) of
+    case matches(BParts, RoutingParts)
+        orelse matches(RoutingParts, BParts)
```

Comment:
i agree on the possible misuse of  `taskd.#` but that was used as an example and the shortcut for cleaninbg can be removed/refactored. so what you're saying is that you can use wildcard as log as it is the first routing part ? that would only service the purposes of crossbar with the version. it's `AMQP-like` not `AMQP-restricted`. i don't see querying `tasks.help.*` as a misuse of kazoo_bindings. how would you query bindings in this use case ?